app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -131,28 +130,21 @@ UserSession getAuthenticatedAndConsentedSession() throws NotAuthenticatedExcepti
         }
         return session;
     }
-
-    /**
-     * Checks if the user is in the "admin" group.
-     */
-    UserSession getAuthenticatedAdminSession() throws NotAuthenticatedException, UnauthorizedException {
-        UserSession session = getAuthenticatedSession();
-        if (!session.getUser().isInRole(ADMIN_GROUP)) {
-            throw new UnauthorizedException();
-        }
-        return session;
-    }
-
-    UserSession getAuthenticatedResearcherSession() throws NotAuthenticatedException, UnauthorizedException {
```

Comment:
Check to make sure you've removed the 3 methods that were role-specific, in favor of the one that takes the role and is more reusable.